tests/data/ericsson-tn_tn.json

```diff
@@ -0,0 +1,7781 @@
+{
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": "<private>",
+                    "sysObjectID": ".1.3.6.1.4.1.193.81.1.1.3",
+                    "sysDescr": "MINI-LINK Traffic Node",
+                    "sysContact": "<private>",
+                    "version": "MINI-LINK_TN_5.4FP_LH_1.6FP_R32B126",
+                    "hardware": "AMM 2p B",
+                    "features": null,
+                    "os": "ericsson-tn",
+                    "type": "wireless",
+                    "serial": null,
```

Comment:
No serial available?

Reply:
My unit is not returning a value on entPhysicalSerialNum.1. I did add it because I hope perhaps other units do. Keep it or remove it?

Reply:
They might have some own. Did u check the mib? Else no problems

Reply:
try walking entPhysicalSerialNum it may be at another index

Reply:
The system I'm polling exists of several components. The indoor unit which I'm querying SNMP to has its serial on ENTITY-MIB::entPhysicalSerialNum.1913126911. But because of the number, I'm a bit afraid that this is different on every other device....

ENTITY-MIB::entPhysicalSerialNum.1913126911 = STRING: XXXXXX

Serials are shown under inventory... so should I just leave it out?

![image](https://user-images.githubusercontent.com/17925078/135461982-7c0846ce-ecd9-43e4-9211-c12621b64bc6.png)

Reply:
Fine with either way, but no point in having a serial entry if it doesn't work ;)